TCMSApp/src/client/app/runs/runs.service.js

```diff
@@ -8,8 +8,16 @@
     RunsApiService.$inject = ['$resource', 'apiUrl'];
 
     function RunsApiService($resource, apiUrl) {
+        var currentRun;
 
         return {
+            currentRun: function(_currentRun) {
+                if (!!_currentRun) {
+                    currentRun = _currentRun;
+                }
+
+                return currentRun;
+            },
             getRuns: function () {
```

Comment:
and also have you thought about the name "getRuns"? it assumes that we are getting some runs. But it is kind of a lie. we are obtaining runResource.